source/ui/dataman/AMImagePropertyEditor.h

```diff
@@ -41,7 +48,6 @@ class AMImagePropertyEditor : public QFrame
 	explicit AMImagePropertyEditor(QWidget *parent = 0);
 	AMImagePropertyEditor(bool useStandardColorMap, int mplotStandardColorMap, const QColor& firstColor, const QColor& secondColor, double brightness, double contrast, double gamma, QWidget* parent = 0);
 	AMImagePropertyEditor(const MPlotColorMap& initialMap, QWidget* parent = 0);
-	virtual ~AMImagePropertyEditor();
```

Comment:
You still want the destructor.  I'll add it back in.  It's good to always have the virtual destructor to ensure proper object cleanup of subclassed objects.